packages/core/src/services/authenticationService.js

```diff
@@ -15,13 +16,13 @@ export const authenticationServiceInternal = WebStorageStateStoreInt => (
   if (userManager) {
     return userManager;
   }
-  const overridenConfiguration = { ...configuration };
+  const overriddenConfiguration = { ...configuration };
 
   if (UserStore) {
-    overridenConfiguration.userStore = new WebStorageStateStoreInt({ store: new UserStore() });
+    overriddenConfiguration.userStore = new WebStorageStateStoreInt({ store: new UserStore() });
   }
-  console.log('overridenConfiguration', overridenConfiguration);
-  userManager = new UserManager(overridenConfiguration);
+  oidcLog.info('overriddenConfiguration', overriddenConfiguration);
```

Comment:
C'est un relicat de dev de ma part. Le console.log ne doit même pas exister. Au pire mettre un debug.

Reply:
Yep, I've changed it to `oidcLog.debug`